docs/conf/portland/2018/news/announcing-speakers.rst

```diff
@@ -0,0 +1,60 @@
+* Bob Watson – `Audience, Market, Product: Tips for strategic API documentation planning <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-bob-watson>`_
+* Brian MacDonald – `Writing the Next Great Tech Book <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-brian-macdonald>`_
+* Camille Acey – `Rewrite the Docs!: Field Notes from the Radical IT department <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-camille-acey>`_
+* Erin Grace – `Document Yourself: Practical Tips for a Low(er)-Stress Portfolio <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-erin-grace>`_
+* Erin McKean – `What Writing Dictionaries Taught Me About Writing Documentation (And What I Had to Unlearn) <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-erin-mckean>`_
+* Havi Hoffman – `Not the docs: Shepherding a developer blog of many voices across many timezones <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-havi-hoffman>`_
+* Jen Lambourne – `Research like you’re wrong: Lessons from user research gone rogue <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-jen-lambourne>`_
+* Kat King – `Building Empathy-Driven Developer Documentation <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-kat-king>`_
+* Larry Ullman – `Making Your Code Examples Shine <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-larry-ullman>`_
+* Neal Kaplan – `Where do I start? The art and practice of documentation triage <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-neal-kaplan>`_
+* Sarah Day – `Starting from Scratch: Finding and Hiring Junior Writers <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-sarah-day>`_
+* Steve Stegelin – `Graphic Content Warning: The Pros, Cons, and Alternatives to Screenshots <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-steve-stegelin>`_
+* Ted Hudek – `Eight Essential Tips for the Enlightened Tech Writer <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-ted-hudek>`_
+* Thursday Bram – `What They Don't Tell You About Creating New Style Guides <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-thursday-bram>`_
+
+You can head over to the `speaker page <http://www.writethedocs.org/conf/portland/2018/speakers/>`_ to see the full abstracts. We hope there’s something there for everyone to enjoy, and also something that will stretch your horizons a little bit, too.
+
+
+Tickets are going fast!
+-----------------------
+If you don't have your ticket yet, now's a great time to snap one up. The conference has sold out for the last several years, so if you're planning to come, don't leave it to the last minute.
```

Comment:
"has sold out" sounds a bit like "omg you sellouts". Maybe "the conference tickets have sold out"?